common/constants.go

```diff
@@ -33,6 +33,10 @@ const NewList PeerType = "new"
 // based on their top up stake
 const AuctionList PeerType = "auction"
 
+// SelectedFromAuctionList represents the list of peers which have been selected from AuctionList based on
+// their top up to be distributed on the WaitingList in the next epoch
+const SelectedFromAuctionList PeerType = "selectedFromAuction"
```

Comment:
"winners" maybe - it is shorter

Reply:
I feel like "winners" is not directly linked to `auction`, although I agree that it is shorter, I think this name better reflects its purpose